Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1574,7 +1568,15 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
   if (mInputSources[GIndex::EMC]) {
     // fill EMC cells to tables
     // TODO handle MC info
-    fillCaloTable(caloEMCCells, caloEMCCellsTRGR, caloCellsCursor, caloCellsTRGTableCursor, bcsMap);
+    auto* caloEventHandler = new o2::emcal::EventHandler<o2::emcal::Cell>();
+    fillCaloTable(caloEventHandler, caloEMCCells, caloEMCCellsTRGR, caloCellsCursor, caloCellsTRGTableCursor, bcsMap, 1);
+    delete caloEventHandler;
```

Comment:
can you avoid new/delete, using instead stack object, i.e.
```
o2::emcal::EventHandler<o2::emcal::Cell> caloEventHandler;
fillCaloTable(&caloEventHandler, caloEMCCells, caloEMCCellsTRGR, caloCellsCursor, caloCellsTRGTableCursor, bcsMap, 1);
```
same for the PHOS. Can be done in different PR, since this one already passed CIs